src/components/SearchInput/SearchInput.module.scss

```diff
@@ -0,0 +1,31 @@
+@import "styles/theme";
+
+.formLayout {
+  margin-top: 30px;
+}
+
+.form {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.input {
+  height: 1.5em;
+  width: 40%;
+}
+
+.searchBtn {
+  margin-left: 0.5em;
+  background-color: white;
+  border: 1.5px solid black;
+  width: 8%;
+  font-weight: bold;
```

Comment:
Same as above, with both the `bold` here, and the `9px` font size below (in theme).